### PR TITLE
CMM 6 cards

### PR DESCRIPTION
**Pull request by Simisays**
Narci's TriggeredCard variable isn't working

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/narci_fable_singer.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Narci, Fable Singer
+ManaCost:1 W B G
+Types:Legendary Creature Human Bard
+PT:3/3
+K:Lifelink
+T:Mode$ Sacrificed | ValidCard$ Enchantment.YouCtrl | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice an enchantment, draw a card.
+SVar:TrigDraw:DB$ Draw
+T:Mode$ AbilityTriggered | ValidMode$ CounterAdded | ValidSource$ Saga.YouCtrl | ValidSpellAbility$ Triggered.LastChapter | TriggerZones$ Battlefield | Execute$ TrigDrain | TriggerDescription$ Whenever the final chapter ability of a Saga you control resolves, each opponent loses X life and you gain X life, where X is that Saga's mana value.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/narci_fable_singer.txt:8]: Checkout Tom Bombadil with
`T:Mode$ AbilityResolves | ValidSource$ Saga.YouCtrl | ValidSpellAbility$ Ability.LastChapter`

File: forge-gui/res/cardsfolder/upcoming/leori_sparktouched_hunter.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Leori, Sparktouched Hunter
+ManaCost:U R W
+Types:Legendary Creature Elemental Cat
+PT:3/3
+K:Flying
+K:Vigilance
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigChoose | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, choose a planeswalker type. Until end of turn, whenever you activate an ability of a planeswalker of that type, copy that ability. You may choose new targets for the copies.
+SVar:TrigChoose:DB$ ChooseType | Type$ Planeswalker | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | Triggers$ TrigAbilityActivated | SpellDescription$ Until end of turn, whenever you activate an ability of a planeswalker of that type, copy that ability. You may choose new targets for the copies.
+SVar:TrigAbilityActivated:Mode$ AbilityCast | ValidCard$ Planeswalker.ChosenType | ValidSA$ SpellAbility | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Until end of turn, whenever you activate an ability of a planeswalker of that type, copy that ability. You may choose new targets for the copies.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/leori_sparktouched_hunter.txt:10]: `ValidSA$ Activated`

File: forge-gui/res/cardsfolder/upcoming/jayas_phoenix.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Name:Jaya's Phoenix
+ManaCost:4 R
+Types:Creature Phoenix
+PT:3/3
+K:Flying
+K:Haste
+T:Mode$ DamageDone | CombatDamage$ True | ValidTarget$ Player | ValidSource$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigDelayedTrigger | TriggerDescription$ Whenever CARDNAME deals combat damage to a player or planeswalker, copy the next loyalty ability you activate this turn when you activate it. You may choose new targets for the copy.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jayas_phoenix.txt:7]: wrong target

File: forge-gui/res/cardsfolder/upcoming/jayas_phoenix.txt [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+SVar:TrigDelayedTrigger:DB$ DelayedTrigger | AILogic$ SpellCopy | Execute$ EffTrigCopy | ThisTurn$ True | Mode$ SpellAbilityCast | ValidSA$ Activated.Loyalty | ValidActivatingPlayer$ You | SpellDescription$ Copy the next loyalty ability you activate this turn when you activate it.
+SVar:EffTrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | Amount$ 1 | MayChooseTarget$ True
+T:Mode$ SpellCast | ValidCard$ Card.Planeswalker | ValidActivatingPlayer$ You | TriggerZones$ Graveyard | Execute$ TrigReturn | TriggerDescription$ Whenever you cast a planeswalker spell, you may return CARDNAME from your graveyard to the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jayas_phoenix.txt:10]: Optional

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/jayas_phoenix.txt]: `OptionalDecider$ True` does not exist

File: forge-gui/res/cardsfolder/upcoming/abstruse_archaic.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Abstruse Archaic
+ManaCost:4
+Types:Creature Avatar
+PT:3/4
+K:Vigilance
+A:AB$ CopySpellAbility | Cost$ 1 T | TgtPrompt$ Select target activated or triggered ability you control from a colorless source | TargetType$ Activated.YouCtrl,Triggered.YouCtrl | ValidTgts$ Card.Colorless | AILogic$ AlwaysCopyActivatedAbilities | MayChooseTarget$ True | SpellDescription$ Copy target activated or triggered ability you control from a colorless source. You may choose new targets for the copy. (Mana abilities can't be targeted.)
+DeckHints:Type$Artifact|Eldrazi
+Oracle:Vigilance{1}, {T}: Copy target activated or triggered ability you control from a colorless source. You may choose new targets for the copy. (Mana abilities can't be targeted.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/abstruse_archaic.txt:8]: linebreak missing

**Review by tool4ever — APPROVED** [review]

**Pull request merged**